src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -821,6 +970,12 @@ public boolean isCourseArchived(CourseAttributes course, InstructorAttributes in
         return result;
     }
     
+    /**
+     * Returns a list of courseIds for all archived courses for all instructors.
```

Comment:
I have discussed this with Dr. Damith and he has mentioned that for simple/straightforward methods where the return value and the description could be very similar, it is not necessary to write a description, but simply write it as the return value. You could keep it as it is if you feel that it is still better to include the method summary in the description.